packages/blade/src/components/Morph/Morph.stories.tsx

```diff
@@ -69,3 +81,105 @@ const MorphTemplate: StoryFn<typeof Morph> = (args) => {
 };
 
 export const Default = MorphTemplate.bind({});
+Default.args = {
+  layoutId: 'button-to-input-morph',
+};
+
+export const MorphOnText = (args: MorphProps): React.ReactElement => {
+  const [showPage, setShowPage] = React.useState(false);
+
+  return (
+    <Box minHeight="400px" height="100%">
+      <AnimatePresence>
+        {showPage ? (
+          <Box textAlign="center">
+            <Box>
+              <Morph {...args}>
+                <Display display="inline-block">Payment Pages</Display>
+              </Morph>
+            </Box>
+
+            <Morph layoutId="subtext">
+              <Text marginTop="spacing.4" display="inline-block">
+                Welcome to payment pages!
+              </Text>
+            </Morph>
+          </Box>
+        ) : (
+          <Box display="flex" alignItems="center" justifyContent="center" height="100%">
+            <Card onClick={() => setShowPage(true)} width="300px">
+              <CardBody>
+                <Morph {...args}>
+                  <Heading display="inline-block">Payment Pages</Heading>
+                </Morph>
+
+                <Morph layoutId="subtext">
+                  <Text marginTop="spacing.4" display="inline-block">
+                    Payment Pages allow you to build pages without writing any code. Click this card
+                    to know more (and see some motion magic)
+                  </Text>
+                </Morph>
+              </CardBody>
+            </Card>
+          </Box>
+        )}
+      </AnimatePresence>
+    </Box>
+  );
+};
+
+MorphOnText.args = {
+  layoutId: 'card-heading',
+};
+
+export const MorphBorderRadius = (): React.ReactElement => {
+  const [showPage, setShowPage] = React.useState(false);
+
+  return (
+    <Box minHeight="400px" height="100%">
+      <Button marginBottom="spacing.5" onClick={() => setShowPage(!showPage)}>
+        Toggle Morph
+      </Button>
+      <AnimatePresence>
+        {showPage ? (
+          <Morph layoutId="box-shape">
```

Comment:
Btw we should be able to morph between any N number of components right? 

This would be important so we can do things like this: https://codesandbox.io/p/sandbox/4uph8w

Reply:
yup this is possible with morph